### PR TITLE
JDF-236 Fixed issues with link traversal in the contributing guide

### PR DESCRIPTION
**Pull request by VineetReynolds**
Removed the use of the simple-expand plugin. The events attached on the enclosing div prevented links from being navigated to.
Replaced the simple-expand plugin with the Bootstrap collapse plugin.

### CODE REVIEW

File: quickstarts/get-involved/index.html.haml [hunk 1]
```diff
@@ -6,11 +6,11 @@ layout: getinvolved
...
 %br(style="clear: left;")
 
-.expand-container
-  %a.expander(href="#")
+.expand-container.collapsed
+  %a.expander{href => "#", "data-toggle" => "collapse", "data-target" => ".content"}
     %strong
       Don't forget to read the Contributing Guide
-  .content(style="display:none; ")
+  .content.collapse{style => "display:none; "}
```
Comment (LightGuard) [inline quickstarts/get-involved/index.html.haml]: Doesn't collapse already have display:none?

**Comment by sgilda** [comment]
I tested this and it works great!

File: quickstarts/get-involved/index.html.haml (hunk 1 above)
Comment (sgilda) [inline quickstarts/get-involved/index.html.haml]: @LightGuard: Where is collapse defined? I will look.

Comment (VineetReynolds) [inline quickstarts/get-involved/index.html.haml]: @LightGuard It doesn't have. But you're right, the style is unnecessary. The plugin modifies the height from 0px to auto, which is good enough.

**Comment by VineetReynolds** [comment]
I pushed in a newer commit without the {style => "display:none;} bit.

**Comment by LightGuard** [comment]
Works for me then.

On Wed, Jun 5, 2013 at 10:02 AM, Vineet Reynolds
notifications@github.comwrote:

> I pushed in a newer commit without the {style => "display:none;} bit.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/jboss-jdf/jdf-site/pull/18#issuecomment-18987541
> .

## 

Jason Porter
http://en.gravatar.com/lightguardjp

**Comment by sgilda** [comment]
Merged.

**Pull request merged**